.github/workflows/ci.yaml

```diff
@@ -87,28 +90,28 @@ jobs:
           GIT_SSH_COMMAND: ssh -o StrictHostKeyChecking=accept-new -o CheckHostIP=no
         run: |
           pipenv run pytest -ra -n auto -v --fulltrace tests
-
-  build:
-    name: Build Package
+  vendor:
+    name: Vendoring
+    needs: tests
```

Comment:
I actually really depend on the vendoring build running in parallel and being one of the first things to complete, when working on patches.  It seems the current actions run doesn't show the vendor step anymore.

Reply:
The current actions moved the vendor step to after the tests. It should be before the tests, if the new code relies on that. I can fix this.